drivers/flash/Kconfig.simulator

```diff
@@ -26,6 +26,14 @@ config FLASH_SIMULATOR_DOUBLE_WRITES
 	 If selected, writing to a non-erased program unit will succeed, otherwise, it will return an error.
 	 Keep in mind that write operations can only pull bits to zero, regardless.
 
+config FLASH_SIMULATOR_NOALLOC
+	bool "No buffer allocated, reads, writes and erase are simulated"
```

Comment:
Only buffer of write block size allocated ...

Reply:
OK, will fix.